concourse/pipelines/linux-image-build.jsonnet

```diff
@@ -235,7 +235,11 @@ local imgpublishjob = {
   trigger:: if tl.env == 'testing' then true
   else false,
 
-  citfilter:: '^(cvm|livemigrate|suspendresume|loadbalancer|guestagent|hostnamevalidation|imageboot|licensevalidation|network|security|hotattach|lssd|disk|packagevalidation|ssh|metadata|vmspec)$',
+  // Use citfilter list as default; append citsuites if not nil
+  // citsuites must be in the valid regex concatenation format of 'item1|item2|item3'
+  citsuites:: '',
```

Comment:
additionalcitsuites would be a more descriptive name

Reply:
Done